php/Blockonomics.php

```diff
@@ -46,7 +48,10 @@ public function new_address($secret, $crypto, $reset=false)
         if($crypto == 'btc'){
             $url = Blockonomics::NEW_ADDRESS_URL.$get_params;
         }else{
-            $url = Blockonomics::BCH_NEW_ADDRESS_URL.$get_params;
+            //Bitcoin payments break if using BCH_NEW_ADDRESS_URL
```

Comment:
> You mentioned that BCH_NEW_ADDRESS_URL makes it so that the bitcoin payments breaks? If so I should probably comment that part out?

Using `$BCH_Enabled = get_option('blockonomics_bch');` inside new_address would have broken functionality. It is unneccessary to comment this out as we did not alter `new_address()`

Reply:
Uncommented Blockonomics::BCH_NEW_ADDRESS_URL.